src/libraries/System.Reflection.Metadata/src/System/Reflection/Metadata/BlobBuilder.cs

```diff
@@ -332,7 +332,7 @@ public void WriteContentTo(Stream destination)
 
             foreach (var chunk in GetChunks())
             {
-                destination.Write(chunk._buffer, 0, chunk.Length);
+                destination!.Write(chunk._buffer, 0, chunk.Length);
```

Comment:
`Throw.ArgumentNull(nameof(destination));` method above should be annotated with `[DoesNotReturn]`, then we will not needed to bang here

Reply:
Looks like I had already annotated the method, so the bang was redundant here.